scripts/docsify-edit-on-github.js

```diff
@@ -1,47 +1,47 @@
-;(function(win) {
-  win.EditOnGithubPlugin = {}
+(function (win) {
+  win.EditOnGithubPlugin = {};
 
   function create(docBase, options) {
-    options = options || {}
-    title = options.title || 'Edit on github'
-    customURLs = options.customURLs || {}
-    var docEditBase = docBase.replace(/\/blob\//, '/edit/')
+    options = options || {};
+    title = options.title || "Edit on github";
+    customURLs = options.customURLs || {};
+    var docEditBase = docBase.replace(/\/blob\//, "/edit/");
 
     function editDoc(event, vm) {
-      var docName = vm.route.file
+      var docName = vm.route.file;
 
       if (docName) {
-        var editLink = customURLs[docName] || docEditBase + docName
-        window.open(editLink)
-        event.preventDefault()
-        return false
+        var editLink = customURLs[docName] || docEditBase + docName;
+        window.open(editLink);
+        event.preventDefault();
+        return false;
       } else {
-        return true
+        return true;
       }
     }
 
-    win.EditOnGithubPlugin.editDoc = editDoc
+    win.EditOnGithubPlugin.editDoc = editDoc;
 
-    return function(hook, vm) {
-      win.EditOnGithubPlugin.onClick = function(event) {
-        EditOnGithubPlugin.editDoc(event, vm)
-      }
+    return function (hook, vm) {
+      win.EditOnGithubPlugin.onClick = function (event) {
+        EditOnGithubPlugin.editDoc(event, vm);
+      };
 
       var header = [
         '<div style="overflow: auto">',
         '<p style="float: right"><a href="',
         docBase,
         '" target="_blank" onclick="EditOnGithubPlugin.onClick(event)">',
         title,
-        '</a></p>',
-        '</div>'
-      ].join('')
+        "</a></p>",
```

Comment:
I need to change this to use single quotes.

A config item 

https://prettier.io/docs/en/configuration.html